src/components/icons/Icon.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
```

Comment:
I don't think I understand what's the purpose of this component. Why can't we just directly pass the icon reference to the dropdown component instead of having deeply locked Icon component?

Reply:
As we discussed on the Slack this component will just get new shape of purpose.